README.md

```diff
@@ -270,6 +270,8 @@ You are ready to deploy:
 $ git push heroku master
 ```
 
+EmberCLI compilation happens at deploy-time, triggered by the `asset:precompilation` rake task.
```

Comment:
Do you think this should be in a more prominent place?

Could you replace `EmberCLI` with `EmberCli`?

Isn't the Rake task `assets:precompile`?